### PR TITLE
feat(building-ranger-with-docker): automatic build release with tag

### PR DESCRIPTION
**Pull request by unknowntpo**
## What changes were proposed in this pull request?

Build Ranger release through tag.

## How was this patch tested?

Github Action.

### CODE REVIEW

**Comment by unknowntpo** [comment]
@xunliu I think this`release-tag` branch is far behind the master branch, could you rebase it ?

File: .github/workflows/publish-ranger-release.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  contents: write
+  packages: write
+
+jobs:
+  build-and-publish-ranger-release:
+    runs-on: ubuntu-latest
+    timeout-minutes: 120
+    env:
+      input_token: ${{ github.event.inputs.token }}
+      secrets_token: ${{ secrets.PUBLISH_DOCKER_TOKEN }}
```
Comment (xunliu) [inline .github/workflows/publish-ranger-release.yml:18]: Do we need to add a token configured in the GitHub?

Comment (unknowntpo) [reply .github/workflows/publish-ranger-release.yml:18]: Sorry, these tokens are unused, I've removed it.

**Review by xunliu — APPROVED** [review]
@unknowntpo Thank you for your contributions.
LGMT

**Pull request merged**